docs/7-vector-search/5-create-vectors/5-amazon-sagemaker.mdx

```diff
@@ -22,9 +22,9 @@ Popular solutions include:
 
 On the main page of SageMaker you'll find the option to `Set up for single user` or `Set up for organization`. These will set up a domain and a quick start user.
 
-<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/1-main-page.png" url="https://aws.amazon.com/pm/sagemaker/" alt="AWS SageMaker" />
-<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/2-setup-started.png" url="https://aws.amazon.com/pm/sagemaker/" alt="AWS SageMaker" />
-<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/3-preparing-domain.png" url="https://aws.amazon.com/pm/sagemaker/" alt="AWS SageMaker" />
+<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/1-main-page.png" url="https://aws.amazon.com/pm/sagemaker/" alt="Amazon SageMaker" />
```

Comment:
Having the same alt text for each screenshot defies the purpose of alt texts. If I'm using a screenreader and I go through 3 screenshots that are all described as "Amazon SageMaker", I won't get any useful information out of that. If the instructions above the screenshot are clear enough, I'd recommend just setting alt="" which will cause the screenreader to ignore the image.

Reply:
They are. At least in my eyes. I usually try to explain well and have the screenshots really just an additional visual help. Which is great, but not necessary. I guess an empty alt text might even be the better approach then. I'll adjust! :)